config/manifests/bases/service-binding-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ spec:
       name: servicebindings.binding.operators.coreos.com
       version: v1alpha1
     - description: Service Binding for Kubernetes. Describes the connection between a Provisioned Service and an Application Projection. Implements Service Binding Specifcation for Kubernetes (https://github.com/k8s-service-bindings/spec).
```

Comment:
(Preview) Service Binding Implementing specification for Kubernetes (https://github.com/k8s-service-bindings/spec). Describes the connection between a Provisioned Service and an Application Projection.